ENIGMAsystem/SHELL/Platforms/General/POSIX/main.cpp

```diff
@@ -1,5 +1,12 @@
 #include "Platforms/General/PFmain.h"
 
+// lmao this has to be included or else
```

Comment:
This source shouldn't even exist; every platform can define its own `main()`, `WinMain()`,`#include<sdl> main()`, etc. No reason to have a four line file with four more lines specific to one user.